dependencies/Boost/gcc_5_no_cxx11.patch

```diff
@@ -0,0 +1,37 @@
+From eec808554936ae068b23df07ab54d4dc6302a695 Mon Sep 17 00:00:00 2001
```

Comment:
[This is the error ](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=197594) I was seeing

When I tried to apply the patch in that first report it failed for some reason that I can't remember.  I found the other patch [here](https://github.com/bitcoin/bitcoin/pull/6280/files) and it fixed my compile errors.  I think we need to update boost anyway pretty soon.  If compile time is the only downside then I'd be inclined to merge this and then delete it when we upgrade boost.